app/controllers/api/v1/merchants/most_items_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Merchants::MostItemsController < ApplicationController
+  def index
+    render json: Merchant.most_items(params[:quantity])
+  end
+end
```

Comment:
Cool, I haven't gotten here so I was wondering if there'd be any issue calling self defined methods in the render json.